screenpy/resolutions/contains_the_item.py

```diff
@@ -24,9 +25,14 @@ class ContainsTheItem:
 
     def describe(self) -> str:
         """Describe the Resolution's expectation."""
-        return f'A sequence containing "{self.item}".'
+        return f"A sequence containing {tostring(self.item)}."
 
-    @beat('... hoping it contains "{item}".')
+    @property
+    def beatmsg(self) -> str:
+        """format string meant for beat msg"""
+        return f"... hoping it contains {tostring(self.item)}."
+
+    @beat("{beatmsg}")
```

Comment:
I'd prefer if this followed the way we do this in other places, like in `Eventually`:
```python
class Eventually:
    ...
    @beat("{} tries to {performable_to_log}, eventually.")
    ...
    def __init__(self, performable: Performable):
        self.performable = performable
        self.performable_to_log = get_additive_description(self.performable)
```

I like it better because then the message for `beat` will be consistently placed above `perform_as` for all the -ables. I wouldn't like having to dig around to find what the message is/where i need to update it. Can we change this to:
```python
@beat('... hoping it contains "{item_to_log}".')
...
    def __init__(self, item: T) -> None:
        self.item = item
        self.item_to_log = tostring(item)
```

Reply:
I'm just going to leave this comment up here about all the other similar `beatmsg` changes below.

Reply:
> I'd prefer if this followed the way we do this in other places, like in Eventually

I generally agree.  When I started looking at this it occurred to me that I would need to create a new attribute for each class for every time we have a variable that would need special string formatting.  Even though in this PR all of the classes only have a singleton, there are other classes that have more than one variable that appear in the `beat`.    It felt a little overkill to create a new attribute just for logging.  

On the flip side, I'm not a fan of having to look in more than one place for the message.  There has to be a better way, right?

Reply:
>  we might be able to remove the describe bit

I'm so glad you said this.  My spidey sense tells me there is a way to build uniformity between the actions and the questions.  Just not sure how.  

For now,  you want to add the extra attributes?

Reply:
Updated.